src/main/java/io/harness/cf/client/api/DefaultApiFactory.java

```diff
@@ -29,6 +30,8 @@ public static DefaultApi create(
       apiClient.setWriteTimeout(writeTimeout);
       apiClient.setBasePath(basePath);
       apiClient.setDebugging(debug);
+      apiClient.setUserAgent("java " + io.harness.cf.Version.VERSION);
+      apiClient.addDefaultHeader("Hostname", InetAddress.getLocalHost().getCanonicalHostName());
```

Comment:
getLocalHost can throw exception, we should handle it? Also shall we use getHostName instead of CanonicalHostName?

Reply:
Have we tested this? I am not sure we will always have this data from localHost

Reply:
Changed to getHostName() and handled its exception